peer-signer/README.md

```diff
@@ -0,0 +1,47 @@
+# Peer Signer
+This tool is intended to verify a potential candidate for testnet validator.
+Package can be split into two parts, `signer` and `verifier`.
+
+
+Signer part will be used by the candidate.
+With this program candidate is going to sign a secret message provided by us to prove they own the p2p secret they claim to own.
+
+
+Verifier part will be used by Aleph Zero team.
+With this program we are going to check whether supplied public key derives to supplied peer id, signature of secret message is correct and this peer is connected and up to date with our network.
+
+## Step one: signature generation by Validator candidate
+To generate signature, candidate needs a generated p2p address.
+For whole process to work they also need to have a node set up with up to date block creation and public p2p port, so it can be added as a peer.
+To generate signature:
+```console
+$ signer --message "secret message" --p2p-secret "some_path_to_p2p_secret/p2p_secret"
```

Comment:
I do not know how you are going to distribute this binary, but if the user has to build it from source then I would put here (and for verifier) information about running `cargo run --bin signer`